### DISCUSSION TITLE
Is I-NUCLEO-NETX supported?

### DISCUSSION THREAD

**Discussion by fkromer** [stm32duino/Arduino_Core_STM32#1512] [Q&A] [answered]
The Nucleo-32 and Nucleo-64 board variants do not have builtin Ethernet. The Ethernet shield [I-NUCLEO-NETX](https://www.st.com/en/evaluation-tools/i-nucleo-netx.html) can be used to extend ethernet connectivity. Is the I-NULCEO-NETX supported for these Nucleo variants?

**Comment by fpistm** [comment, marked as answer]
Hi @fkromer 
I never use it.
I've searched for an Arduino library for NETX and found nothing.

**Reply by fkromer** [reply]
@fpistm Thanks for letting me know.
